File: 2563.c
#include <stdio.h>

int main(void) {

  int arr[100][100] = {0,};

  int n, a, b, count = 0;
  scanf("%d", &n);

  for(int i = 0; i < n; i++) {
    scanf("%d %d", &a, &b);
    for(int i=b; i<b+10; i++) {
      for(int j=a; j<a+10; j++) {
        arr[i][j] = 1;
      }
    }
  }

  for(int i = 0; i < 100; i++) {
    for(int j=0; j<100; j++) {
      if(arr[i][j] == 1)
        count++;
    }
  }

  printf("%d\n", count);
}

File: 2563.py
#arr = [[0 for col in range(10)] for row in range(10)]
#arr = [[0] * 100 for _ in range(100)]
arr = [[0 for col in range(100)] for row in range (100)]

n = int(input())
count = 0

for k in range(n):
    a, b = map(int, input().split())
    for i in range(a, a+10):
        for j in range(b, b+10):
            arr[i][j] = 1


for i in range(0, 100):
    for j in range(0, 100):
        if arr[i][j] == 1:
            count+=1
        
print(count)
